### PR TITLE
Updating OVA Documentation

### PR DESCRIPTION
**Pull request by VenkyOS**

### CODE REVIEW

File: doc/CORTX_on_Open_Virtual_Appliance.rst [hunk 1]
```diff
@@ -217,8 +217,16 @@ Restart CORTX
 2. Login to the CORTX OVA as cortx and run the following.
 
    - **sudo su -**
+   
+3. Restart openldap and s3 auth server services by the below mentioned commands.
```
Comment (johnbent) [inline doc/CORTX_on_Open_Virtual_Appliance.rst:221]: Why are these hard-coded?  They should be #. and let RST do the dynamic numbering.

File: doc/Importing_OVA_File.rst [hunk 2]
```diff
@@ -166,6 +166,10 @@ If you are facing errors due to usage of old versions, perform the below mention
 
 #. Return to `CORTX on OVA <CORTX_on_Open_Virtual_Appliance.rst>`_, and resume following the instructions there.
 
+Click `here <https://kb.vmware.com/s/article/80467>`_ to navigate to the relevant page and resolve the below mentioned error.
```
Comment (johnbent) [inline doc/Importing_OVA_File.rst:169]: Why does it it say 'Click here to .... resolve the below mentioned error.  Will everyone see this error every single time?  Or only sometimes?  If everyone sees it every time, then put in the instructions to fix it.  If it only sometimes happen, then say, "If you see the below error, then . . . "

**Review by johnbent — CHANGES_REQUESTED** [review]

**Review by sachitanands — APPROVED** [review]

**Review by mukul-seagate11 — APPROVED** [review]
looks good as suggested

File: doc/Importing_OVA_File.rst (hunk 2 above)
Comment (VenkyOS) [reply doc/Importing_OVA_File.rst]: I have asked @mukul-seagate11 and @sachitanands for updates for the same. I will update the same once I get clarity.

File: doc/CORTX_on_Open_Virtual_Appliance.rst (hunk 1 above)
Comment (VenkyOS) [reply doc/CORTX_on_Open_Virtual_Appliance.rst:221]: It will be corrected.

Comment (VenkyOS) [reply doc/CORTX_on_Open_Virtual_Appliance.rst]: Updated and Resolved.

**Pull request merged**